.github/workflows/conan-package.yml

```diff
@@ -0,0 +1,152 @@
+---
+name: conan-package
+
+on:
+  push:
+    paths:
+      - 'src/**'
+      - 'cmake/**'
+      - 'tests/**'
+      - 'conanfile.py'
+      - 'CMakeLists.txt'
+      - '.github/workflows/conan-package.yml'
+    branches:
+      - main
+      - master
+      - dev
+      - 'release/**'
+      - '!15.*'
+      - '![1-4].[0-9]+'
+    tags:
+      - '![1-4].[0-9]+*'
+      - '!Arachne*'
+      - '!ancient*'
+      - '!jedi*'
+
+  pull_request:
+    paths:
+      - 'src/**'
+      - 'cmake/**'
+      - 'tests/**'
+      - 'conanfile.py'
+      - 'CMakeLists.txt'
+      - '.github/workflows/conan-package.yml'
+    branches:
+      - main
+      - master
+      - dev
+      - 'release/**'
+
+env:
+  CONAN_USER: ${{ secrets.CONAN_USER }}
+  CONAN_PASS: ${{ secrets.CONAN_PASS }}
+
+jobs:
+  get-semver:
```

Comment:
Some comments as to why we need this sem-ver job would be nice (i mean, i now know, but we all know that future us will have forgotten it!)